requirements.in

```diff
@@ -7,4 +7,5 @@ pytest
 pytest-cov
 Sphinx
 tox
-twine
\ No newline at end of file
+twine
+hypothesis
```

Comment:
maybe put this back in alphabetical order?

Reply:
Done